common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectoryOAuth2Strategy.java

```diff
@@ -246,4 +254,16 @@ protected TokenResult getTokenResultFromHttpResponse(final HttpResponse response
         return result;
     }
 
+    @Override
+    protected void validateTokenResponse(AzureActiveDirectoryTokenRequest request,
+                                         AzureActiveDirectoryTokenResponse response) {
+        // TODO
+    }
+
+    @Override
+    public String getAuthorizationHeader(AbstractAuthenticationScheme scheme,
+                                         TokenResponse tokenResponse) {
+        return tokenResponse.getAccessToken();
```

Comment:
nit scheme + " " + accessToken?

Reply:
Resolved in cd5512193ebbd75b29ddb77ac9dcf1a6e7dd6b93